go.mod

```diff
@@ -12,21 +12,24 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spinkube/spin-operator v0.1.0
 	github.com/stretchr/testify v1.9.0
-	k8s.io/api v0.29.2
-	k8s.io/apimachinery v0.29.2
+	helm.sh/helm/v3 v3.14.3
+	k8s.io/api v0.29.3
```

Comment:
Why is this included if it isn't a direct dependency of the plugin?

Reply:
If you're talking about the `helm.sh/helm/v3` dependency, it's added because I decided to use `chart` and `chartutil` from helm itself instead of rolling everything manually